lntest/node.go

```diff
@@ -262,7 +269,8 @@ func (hn *HarnessNode) start(lndError chan<- error) error {
 	// If the logoutput flag is passed, redirect output from the nodes to
 	// log files.
 	if *logOutput {
-		fileName := fmt.Sprintf("output%d.log", hn.NodeID)
+		fileName := fmt.Sprintf("output-%d-%s-%s.log", hn.NodeID,
+			hn.cfg.Name, hex.EncodeToString(hn.PubKey[:logPubKeyBytes]))
```

Comment:
The node's public key hasn't yet been instantiated at this point. If you look at the travis run, you'll see it's all zeroes for most of the nodes. I think adding the public key is definitely a good idea, but it may need to be done further down the line, as we actually don't know the public key of the node until it starts...

Reply:
Ah, I was wondering about that. I thought that most IDs were all zeroes because the integration tests were set up this way (since some IDs were initialized).
Didn't occur to me it just might be a timing problem.

I'll look into it.